src/app/page/Missions/Missions.jsx

```diff
@@ -39,9 +39,12 @@ const MissionsPage = ({ user, history, firebase, ...rest }) => {
       User.assignAsVolunteer(firestore, missionId, user.uid);
     }
   }
-
-  if (!missions) {
-    return <div> isloading...</div>;
+  if (missions === undefined) {
```

Comment:
Any particular reason for doing explicit check against undefined versus truthy check? Moreover, isn't missions an array? Would it be better to check for length then?

Reply:
Missions is undefined before data comes in, so I believe checking the length property would not work.

Reply:
Gotcha!